turnkey-sysinfo

```diff
@@ -12,17 +12,15 @@
 #
 import os
 import re
-
-from libsysinfo import disk
-from libsysinfo.memstats import MemoryStats
-
-from datetime import datetime
-import time
 import subprocess
+import time
+from datetime import datetime
 
 import netinfo
+from libsysinfo import disk
+from libsysinfo.memstats import MemoryStats
 
-NIC_BLACKLIST = ('lo')
+NIC_BLACKLIST = "lo"
```

Comment:
This change is incorrect. It should be am iterable containing interface names. https://github.com/turnkeylinux/turnkey-sysinfo/blob/9ad01d3c7dab93b760491e8017b64ac9ce536fe2/turnkey-sysinfo#L30